src/lualib/ArrayIncludes.ts

```diff
@@ -0,0 +1,22 @@
+// https://www.ecma-international.org/ecma-262/9.0/index.html#sec-array.prototype.includes
+function __TS__ArrayIncludes<T>(this: T[], searchElement: T, fromIndex = 0): boolean {
+    const len = this.length;
+    let k = fromIndex;
+
+    if (fromIndex < 0) {
+        k = len + fromIndex;
+    }
+
+    if (k < 0) {
+        k = 0;
+    }
+
+    while (k < len) {
+        if (this[k] === searchElement) {
```

Comment:
FWIW `.includes` uses a different comparison algorithm

```js
[NaN].indexOf(NaN) // => -1
[NaN].includes(NaN) // => true
```

But I'm okay with leaving it as an edge case